src/main/java/calculator/AddCalculator.java

```diff
@@ -0,0 +1,23 @@
+package calculator;
+
+import java.util.Arrays;
+
+public class AddCalculator {
+    private final String[] stringArr;
+    private int sum;
+
+    public AddCalculator(String[] strArr) {
+        this.stringArr = strArr;
+    }
+    public int add() {
+        int[] toIntArr = convertStringToInt();
+        for (int i : toIntArr) {

```

Comment:
네이밍을 더 가독성 좋게 바꾸는 것도 좋을 것 같아요!
```suggestion
        for (int number : numbers) {

```

이런식으로요!

Reply:
핫...부끄럽네욤... 습관적으로 `i`를 쓰는것같아요..! 
앞으론 좀 더 가독성 좋게 바꾸도록 하겠습니다!!